{{ cookiecutter.project_shortname }}/run-tests.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env sh
 {% include 'misc/header.py' %}
 
-pydocstyle {{ cookiecutter.package_name }} && \
+pydocstyle {{ cookiecutter.package_name }} tests && \
 isort -rc -c -df **/*.py && \
-check-manifest --ignore ".travis-*" && \
+check-manifest --ignore ".travis-*,docs/_build" && \
```

Comment:
I think it's an issue with the naming of requirements-builder output:
``requirements-builder --level=pypi setup.py > .travis-release-requirements.txt``, this command has nothing to do with travis, as in principle developer might be interested in testing the module locally, using requirements-builder to compile different levels of requirements. In such a case he will have those ".travis-FOO.txt" files within his local repo too. A bettter name for the output of this file would be something Travis/CI-agnostic, e.g.: "built-requirements-release.txt"